update-translations.sh

```
@@ -396,8 +375,34 @@
     fi
 
     # Push the branch to GitHub
-    log "Pushing changes to origin ($FORK_REPO_NAME)"
-    git push origin "$BRANCH_NAME"
+    if git remote | grep -qx 'origin'; then
+      log "Pushing changes to origin ($FORK_REPO_NAME)"
+      # Before pushing, derive the owner from the 'origin' remote URL.
+      # This ensures the user in the PR head matches the push destination.
+      local fork_owner
```

Shellcheck (0.10.0)

[error] 'local' is only valid in functions.

(SC2168)